app/views/layouts/find_layout.html.erb

```diff
@@ -4,7 +4,10 @@
     <title><%= yield :page_title %> - <%= t("service_name.find") %> - GOV.UK</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-    <%= canonical_tag %>
+
+    <% url = URI(request.url.split("?").first) %>
```

Comment:
I didn't test but if the url has anchors, will work this line?

Reply:
I think we could use `request.base_url + request.path` instead. Let me try and see.

Reply:
Yes, using `request.base_url + request.path` might work